### PR TITLE
javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by somayehdvr**

### CODE REVIEW

File: javascript/javascript1/week2/weather.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+function weatherWear(temperature){
+    let clothesToWear ={
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/weather.js]: Cool use of object and arrays! 
 Be aware that item indexes of an array might change. So unless the items and their order are a constant, you might run into trouble further down the line.

File: javascript/javascript1/week2/student.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        console.log("please enter your name")
+        return
+    }
+    else{
+        if(class07Students.length < 6 || studentName==("Queen")){
+            if (class07Students.includes(studentName)){
+                console.log(`Student ${studentName} is already in the class`)
+                return
+            }
+            else{
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student.js]: redundant else

File: javascript/javascript1/week2/student.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+function addStudentToClass(studentName) {
+    if (!studentName) {
+        console.log("please enter your name")
+        return
+    }
+    else{
+        if(class07Students.length < 6 || studentName==("Queen")){
+            if (class07Students.includes(studentName)){
+                console.log(`Student ${studentName} is already in the class`)
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student.js:9]: could we not check this before?

File: javascript/javascript1/week2/student.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                return
+            }
+            else{
+                class07Students.push(studentName)
+                return
+            }
+        }
+        else{
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student.js]: this whole if chain could be simplified.  consider how 🤔

File: javascript/javascript1/week2/student.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+addStudentToClass("Sam");
+addStudentToClass("Erik");
+addStudentToClass("Elena");
+addStudentToClass("Joe");
+addStudentToClass("Erik"); //Add a student that is already in the class
+addStudentToClass("Sergey");
+addStudentToClass("John");
+addStudentToClass(""); //Add empty string
+addStudentToClass("Nathan"); //Add more students than there is space for
+addStudentToClass("Queen"); //Add the Queen to a full class
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student.js:36]: bonus question: "what happens if i add a number or a boolean value?"

File: javascript/javascript1/week2/flight.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+function getFullname (firstname, surname, useFormalName, gender) {
+    if (!firstname || !surname) {
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/flight.js:2]: if else sequence could be simplified

File: javascript/javascript1/week2/event.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+function getEventWeekday(daysFromNow) {
+    const weekdays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const today = new Date();
+    const date = (today.getDay() + daysFromNow) % 7
+    return weekdays[date];
+}
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/event.js:6]: good job!

**Review by FrederikNygart — COMMENTED** [review]
Good job! a few comments, hit me up on slack if you need my help 🤙

File: javascript/javascript1/week2/weather.js (hunk 1 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/weather.js]: So I changed it to const

File: javascript/javascript1/week2/flight.js (hunk 6 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/flight.js:2]: I made it nested

File: javascript/javascript1/week2/student.js (hunk 3 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/student.js:9]: Done

File: javascript/javascript1/week2/student.js (hunk 4 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/student.js:19]: this was for the first if

File: javascript/javascript1/week2/student.js (hunk 2 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/student.js]: This was for the first else

File: javascript/javascript1/week2/student.js (hunk 4 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/student.js]: I've made some changes

File: javascript/javascript1/week2/student.js (hunk 5 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/student.js:36]: It adds them to the class

File: javascript/javascript1/week2/event.js (hunk 7 above)
Comment (somayehdvr) [reply javascript/javascript1/week2/event.js:6]: Thanks

**Review by FrederikNygart — APPROVED** [review]

**Pull request merged**